### PR TITLE
Added missing enum values

### PR DESCRIPTION
**Pull request by andyatwebscale**

### CODE REVIEW

**Comment by SidneyAllen** [comment]
@jenksguo and @GraceYBR - FYI I have an change in my branch which adds a missing enum.  I plan include this in the next release of the OpenAPI spec.

https://github.com/XeroAPI/Xero-OpenAPI/commit/d9b4e601f8596d83c77e072224fc4c191297786e

**Comment by andyatwebscale** [comment]
Thats great news @SidneyAllen, thank you. Xero turn off OAuth1.0 really soon and we're testing our OAuth2 connection as we speak. I _think_ this is the only blocker.

**Comment by andyatwebscale** [comment]
@SidneyAllen, will the next release just include a single payment type? If you look at the pull request I've added 6.

**Comment by SidneyAllen** [comment]
@andyatwebscale - if you've identified 6 enums that are missing, please share them in a comment.  Just to be clear, our OpenAPI spec does not modify our actual APIs, but are used as blueprints for building the SDKs.  Enums are a challenge as product teams may add new ones and not notify us of the addition.  Appreciate your help.

**Comment by andyatwebscale** [comment]
Sure, my apologies! These are the ones that we believe are missing (and you can find in EarningsRate.cs)

           `/// <summary>
            /// Enum StatutoryAdoptionPayNonPensionable for value:
            /// </summary>
            [EnumMember(Value = "Statutory Adoption Pay (Non Pensionable)")]
            StatutoryAdoptionPayNonPensionable = 16,

            /// <summary>
            /// Enum StatutoryMaternityPayNonPensionable for value:
            /// </summary>
            [EnumMember(Value = "Statutory Maternity Pay (Non Pensionable)")]
            StatutoryMaternityPayNonPensionable = 17,

            /// <summary>
            /// Enum StatutoryPaternityPayNonPensionable for value:
            /// </summary>
            [EnumMember(Value = "Statutory Paternity Pay (Non Pensionable)")]
            StatutoryPaternityPayNonPensionable = 18,

            /// <summary>
            /// Enum StatutoryParentalBereavementPayNonPensionable for value:
            /// </summary>
            [EnumMember(Value = "Statutory Parental Bereavement Pay (Non Pensionable)")]
            StatutoryParentalBereavementPayNonPensionable = 19,

            /// <summary>
            /// Enum StatutorySharedParentalPayNonPensionable for value:
            /// </summary>
            [EnumMember(Value = "Statutory Shared Parental Pay (Non Pensionable)")]
            StatutorySharedParentalPayNonPensionable = 20,

            /// <summary>
            /// Enum StatutorySickPayNonPensionable for value:
            /// </summary>
            [EnumMember(Value = "Statutory Sick Pay (Non Pensionable)")]
            StatutorySickPayNonPensionable = 21`

**Comment by andyatwebscale** [comment]
Hey @SidneyAllen, the comments above don't actually contain the offending article... Plse check the int values for each too

           ```
            /// <summary>
            /// Enum StatutorySickPayNonPensionable for value: StatutoryBereavementPay
            /// </summary>
            [EnumMember(Value = "Statutory Bereavement Pay")]
            StatutoryBereavementPay = 16, 
```

**Comment by SidneyAllen** [comment]
@andyatwebscale - I've added the enums to our OpenAPI spec version 2.9.4
https://github.com/XeroAPI/Xero-OpenAPI/releases/tag/2.9.4

@jenksguo - I'll leave this PR open for you to evaluate.  There is a unit test, but the code to add the enums will be overwritten by the generated models.

File: Xero.NetStandard.OAuth2/Model/PayrollUk/EarningsRate.cs [hunk 1]
```diff
@@ -125,8 +119,43 @@ public enum EarningsTypeEnum
...
+            /// </summary>
+            [EnumMember(Value = "Statutory Shared Parental Pay (Non Pensionable)")]
+            StatutorySharedParentalPayNonPensionable = 20,
+
+            /// <summary>
+            /// Enum StatutorySickPayNonPensionable for value:
+            /// </summary>
+            [EnumMember(Value = "Statutory Sick Pay (Non Pensionable)")]
+            StatutorySickPayNonPensionable = 21
         }
```
Comment (SerKnight) [inline Xero.NetStandard.OAuth2/Model/PayrollUk/EarningsRate.cs:159]: Hi @andyatwebscale - Making sure you are aware these are now available in `3.17.0`

https://github.com/XeroAPI/Xero-NetStandard/blob/master/Xero.NetStandard.OAuth2/Model/PayrollUk/EarningsRate.cs

They are part of a larger PR as described in the release notes, but this contribution was much appreciated.. 

DM me on twitter for some Xero swag if your keen!
> https://twitter.com/serknight_

**Comment by SerKnight** [comment]
@andyatwebscale - the int values are different as the codegen adds them at a unique index but they are now there.

**Pull request closed without merging**